features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/WireframeMapper.kt

```diff
@@ -20,6 +20,10 @@ import com.datadog.android.sessionreplay.model.MobileSegment
  * Session Replay representation for a specific View type you can implement this on your end.
  */
 interface WireframeMapper<in T : View, out S : MobileSegment.Wireframe> {
+    /**
+     * Determines the behavior during the traversal of the view tree.
+     * @see TraversalStrategy
+     */
     val traversalStrategy: TraversalStrategy
```

Comment:
I think last time I discussed with @louiszawadzki I mentioned that this should not be part of the `WireframeMapper` interface. Instead we could make it part only of the `NonExhaustiveTypeMapper` and check in the traversal strategy if the mapper is of this type in order to resolve the strategy.

Reply:
Yes I agree with @mariusc83, sorry if the requirements were unclear. 
The idea is to leave the `WireframeMapper` interface untouched, and only touch `NonExhaustiveTypeMapper`.

Reply:
I've made this change - now `TraversalStrategy` is internal within `TreeViewTraversal` (it doesn't seem possible to make it private due to usage within the `TraversedTreeView` data class)